### PR TITLE
SieveOfEratosthenes added

### PR DESCRIPTION
**Pull request by durid17**
Implementation of Sieve of eratosthenes added, function gets int n and return all prime numbers before n

### CODE REVIEW

**Comment by durid17** [comment]
Errors are not caused by my code.

File: math/SieveOfEratosthenes.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+	for p := 2; p <= n; p++{ 
+		if isPrime[p]{
+			primes = append(primes, p)
+		}
+	}
+	return primes
+}
+
+/*
```
Comment (nomisrevol) [inline math/SieveOfEratosthenes.go:28]: Why did you comment `func main`? Should it be removed or enable for testing?

File: math/SieveOfEratosthenes.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
+package main
+
+/*
+*gets int n and return all prime numbers before n
```
Comment (nomisrevol) [inline math/SieveOfEratosthenes.go:4]: Please provide more informative description.

**Comment by tjgurwara99** [comment]
Good effort. However, our [contribution guidelines](https://github.com/TheAlgorithms/Go/blob/d2f12849df76203d80762ceca5105531d3424384/CONTRIBUTING.md) have changed. Please take a look at our new [contribution guidelines](https://github.com/TheAlgorithms/Go/blob/d2f12849df76203d80762ceca5105531d3424384/CONTRIBUTING.md) and make changes accordingly. Currently this PR is not up to standard. If no changes are made in the next two weeks, I will consider this PR to be stale and will close this PR then.

**Pull request closed without merging**